vasttools/utils.py

```diff
@@ -315,6 +316,33 @@ def build_SkyCoord(catalog: pd.DataFrame) -> SkyCoord:
     return src_coords
 
 
+def read_selavy(selavy_path: str, cols: list = None) -> pd.DataFrame:
```

Comment:
```suggestion
def read_selavy(selavy_path: str, cols: Optional[List[str]] = None) -> pd.DataFrame:
```
I think this argument is a list of strings? You'll need to import `Optional` and `List` if not already in.